src/components/ContextMenu.css

```diff
@@ -7,28 +7,24 @@
   user-select: none;
   margin: -0.25rem 0 0 0.125rem;
   padding: 0.25rem 0;
-  background-color: #f2f2f2;
-  border: 1px solid #bcbcbc;
-}
-
-.context-menu__option {
```

Comment:
This actually results in two UI regressions:

1. it's too bunched together:
     
     ![image](https://user-images.githubusercontent.com/5153846/73124372-e6ec9800-3f9a-11ea-811a-ab39981c73e1.png)

2. the position calculation doesn't work when there's no min width:

      ![image](https://user-images.githubusercontent.com/5153846/73124383-0edbfb80-3f9b-11ea-971d-1e12f15ebeb3.png)

So to ensure the items don't wrap, you'll prob need to calculate the min width during open, before calculating the position.

Reply:
I'll check those.. Thanks for testing

Reply:
Fixed the width issue.. check it again.

Reply:
Yep, working. That's actually a simple and elegant solution.

Reply:
WTBS, I'd still add `padding` to emulate the previous style.